http://sources.redhat.com/cgi-bin/cvsweb.cgi/libc/linuxthreads/sysdeps/unix/sysv/linux/bits/sigthread.h.diff?r1=1.5&r2=1.6&cvsroot=glibc

Fixes 

../linuxthreads/sysdeps/unix/sysv/linux/bits/sigthread.h:35: error: storage class specified for parameter `type name'

Rediffed to match glibc-2.1.3

--- glibc-2.1.3/linuxthreads/sysdeps/unix/sysv/linux/bits/sigthread.h.orig	1998-09-12 14:33:14.000000000 -0700
+++ glibc-2.1.3/linuxthreads/sysdeps/unix/sysv/linux/bits/sigthread.h	2004-03-04 15:50:30.000000000 -0800
@@ -32,6 +32,6 @@
 				 __sigset_t *__oldmask));
 
 /* Send signal SIGNO to the given thread. */
-extern int pthread_kill __P ((pthread_t __thread, int __signo));
+extern int pthread_kill __P ((pthread_t __threadid, int __signo));
 
 #endif	/* bits/sigthread.h */
